From 252f44c072d3447a32127fc9afe5d0d40f3c510e Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 26 Jan 2011 13:20:20 +0100
Subject: [PATCH 3/6] force 'plain' mode

The new DSS mode breaks XV, so force plain mode

Upstream-Status: Pending

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/omapfb-driver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/omapfb-driver.c b/src/omapfb-driver.c
index 018e040..48aa09c 100644
--- a/src/omapfb-driver.c
+++ b/src/omapfb-driver.c
@@ -326,7 +326,7 @@ OMAPFBPreInit(ScrnInfoPtr pScrn, int flags)
 	OMAPFBProbeController(ofb->ctrl_name);
 
 	/* Do we have the DSS kernel API? */
-	if (stat(SYSFS_DSS_DIR, &st) == 0) {
+	if (0) { //stat(SYSFS_DSS_DIR, &st) == 0) {
 		ofb->dss = TRUE;
 	} else {
 		ofb->dss = FALSE;
-- 
1.8.0

